README.md

```diff
@@ -40,6 +41,20 @@ There are few available subspecs:
 - `FirebaseCrashlytics` - Core + Firebase Crashlytics logger destination;
 - `Complete` - Includes all available functionality. This subspec will be used by default.
 
+### Swift Package manager
+
+This SDK is available through Swift Package Manager. To install it, import it from the following url:
+```
+https://github.com/okta/okta-logger-swift.git
+```
+There are few available products available to import:
+
+- `OktaLoggerCore` - Contains only basic functionality and Console logger destination;
+- `OktaFileLogger` - Core + File logger destination;
+- `OktaFirebaseCrashlytics` - Core + Firebase Crashlytics logger destination;
+- `OktaInstabugLogger` - Core + Instabug logger destination;
+- `OktaLogger` - Includes all available functionality;
```

Comment:
Is there a way to have subspecs the way we do with cocoapods?

Reply:
Yes. In the SPM world seems they're called `Products` and they're defined similarly. If you take a look at the Package.swift file I maintained the same structure. So if we want to import just a specific product you can do so. (e.g. `import LoggerCore` will only import the Core files without any other dependency)